### PR TITLE
log an info if we get multiple departues on the cr instead of a warning

### PR DESCRIPTION
**Pull request by sghiaa**
#### Summary of changes
**Asana Ticket:** [🐛multiple updates error ](https://app.asana.com/0/584764604969369/1138809250698897)

make it an info if its cr

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)

### CODE REVIEW

File: test/prediction_analyzer/vehicle_positions/comparator_test.exs [hunk 1]
```diff
@@ -20,11 +20,24 @@ defmodule PredictionAnalyzer.VehiclePositions.ComparatorTest do
     trip_id: "trip1",
     route_id: "route1",
     direction_id: 0,
-    current_status: :IN_TRANSIT_TO,
+    current_status: :in_transit_to,
```
Comment (lemald) [inline test/prediction_analyzer/vehicle_positions/comparator_test.exs]: Was this change intentional? It looks like the value is still `:IN_TRANSIT_TO` in other places.

Comment (sghiaa) [reply test/prediction_analyzer/vehicle_positions/comparator_test.exs]: whoops, i dont know what command causes that but its a typo of when im trying to copy.

File: test/prediction_analyzer/vehicle_positions/comparator_test.exs [hunk 2]
```diff
@@ -104,6 +117,124 @@ defmodule PredictionAnalyzer.VehiclePositions.ComparatorTest do
...
+      log =
+        capture_log([level: :warn], fn ->
+          Comparator.compare(new_vehicles, old_vehicles)
+        end)
+
+      assert log =~ "One departure, multiple updates"
+    end
+
+    test "does not log an error when there are multiple updates for a commuter rail vehicle" do
+      vehicle = %{@vehicle | route_id: "CR-Fitchburg"}
```
Comment (lemald) [inline test/prediction_analyzer/vehicle_positions/comparator_test.exs]: Did you mean to use `@cr_vehicle` here? It looks like `@cr_vehicle` is currently unused.

Comment (sghiaa) [reply test/prediction_analyzer/vehicle_positions/comparator_test.exs]: at some point i had intended to use it but its unneeded, so ill take it out.
